### PR TITLE
Adventive amp-ad type

### PR DESCRIPTION
**Pull request by Adventive**
Submitting initial PR for amp-ad implementation on behalf of Adventive.

Fulfilling #12481 

Note:
We are developing our ads to use native amp elements, reuse JS between ads, and reduce ad requests. As that is not ready yet, but would like to begin incorporating amp-analytics and other a4a aspects with our current implementation (and new implementation), we have written two ways to consume our amp-ad type; by setting the `isDev`flag

The API's for `renderStart` and `noContentAvailable` have been implemented in both our current and amp-style implementation but that has not been pushed to production yet.

### CODE REVIEW

**Comment by Adventive** [comment]
I signed it!

**Comment by Adventive** [comment]
I signed it!

**Comment by Adventive** [comment]
I signed it!

**Comment by Adventive** [comment]
I signed it!

**Comment by Adventive** [comment]
I've signed the CLA but for some reason it does not show up on file.
I have all emails added to this Github account. 
I made the commit with the same email I signed it with.
I'm unsure what to do.

screen shot 2018-01-11 at 10 49 27 am
screen shot 2018-01-11 at 10 49 53 am
screen shot 2018-01-11 at 10 50 45 am

**Comment by Adventive** [comment]
I signed it!

**Comment by Adventive** [comment]
I signed it!

File: ads/_config.js [hunk 1]
```diff
@@ -187,6 +187,19 @@ export const adConfig = {
...
   },
 
+  adventive: {
+    // prefetch: 'https://ads.adventive.com/ad.js',
+    preconnect: [
+      'https://ads.adventive.com',
+      'https://amp.adventivedev.com',
+    ],
+    renderStartImplemented: true,
+    // clientIdScope: "",
```
Comment (lannka) [inline ads/_config.js]: can you remove ?

Comment (Adventive) [reply ads/_config.js]: It is now removed.

**Comment by Adventive** [comment]
@lannka anymore changes needed?

**Pull request merged**